app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: Player) -> int:
```

Comment:
The parameter `players` should be a list of `Player` objects, not a single `Player`. Change the type annotation to `List[Player]` and ensure you import `List` from `typing`.